assets/css/app.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: 'Poiret One', cursive;
+  font-size 1.6em
+}
+
+.form__button--submit {
+  width: 15%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 0.5em;
+}
+
+.filters__button {
+  text-align: center
+}
+
+.filters__button--active   {
+  width: 15%;
+}
+
+.filters__button--all {
+  width: 15%;
+}
+
+.filters__button--complete {
+  width: 15%;
+}
+
+.total {
+  text-align: center;
+  font-size: 1.5em;
+  font-weight: bolder;
+  color: white;
+  background-color: #337ab7
+}
+
+#task {
+  width: 100%;
+  margin-bottom: 0.5em;
+  text-align: center;
+  font-size: 1.4em;
+}
```

Comment:
Avoid both styling using `#` *and* `.` as the specificity gets messy (one overrides the other etc). Stick to one selector where possible - so here a `.task` selector on each task would make sense.